src/main/java/de/zalando/aruha/nakadi/config/JsonConfig.java

```diff
@@ -37,6 +39,8 @@ public ObjectMapper jacksonObjectMapper() {
         objectMapper.registerModule(enumModule());
         objectMapper.registerModule(new Jdk8Module());
         objectMapper.registerModule(new ProblemModule());
+        objectMapper.registerModule(new JodaModule());
+        objectMapper.configure(WRITE_DATES_AS_TIMESTAMPS , false);
```

Comment:
Do you think that we need any additional tests for this? Or is it already covered by current tests?